src/Decoder.c

```diff
@@ -3082,8 +3082,8 @@ static void ZydisSetAVXInformation(ZydisDecoderContext* context,
             // Nothing to do here
             break;
         case ZYDIS_MVEX_FUNC_RC:
-            instruction->avx.rounding.mode = ZYDIS_ROUNDING_MODE_RN + instruction->raw.mvex.SSS;
-            break;
+            instruction->avx.rounding.mode = ZYDIS_ROUNDING_MODE_RN + (instruction->raw.mvex.SSS & 3);
+            // Intentional fallthrough
```

Comment:
Fyi: We have a ZYAN_FALLTHROUGH macro for this purpose 😃

If you don't want to edit your PR, I can as well change that line after merging.

Reply:
I knew I've seen this macro somewhere. Actually it turns out it's used exactly everywhere except the place I used for reference when making this PR :grinning: Will update both.